legion/robot/legion/robot/profiler_loader.py

```diff
@@ -14,92 +14,66 @@
 #    limitations under the License.
 #
 """
-Variables loader (from profiles/{env.PROFILE}.yml and /{env.CREDENTIAL_SECRETS}.yml files)
+Variables loader from json Cluster Profile
 """
 
 import os
-import typing
-
-import yaml
+import json
 from legion.robot.libraries import dex_client
 from legion.robot.libraries.dex_client import init_session_id
 
-PROFILE_ENVIRON_KEY = 'CLUSTER_NAME'
-PATH_TO_PROFILES_DIR = 'PATH_TO_PROFILES_DIR'
-PATH_TO_COOKIES_FILE = 'PATH_TO_COOKIES'
-CREDENTIAL_SECRETS_ENVIRONMENT_KEY = 'CREDENTIAL_SECRETS'
-
+CLUSTER_PROFILE = 'CLUSTER_PROFILE'
 
-def get_variables(arg: typing.Optional[str] = None, profile: typing.Optional[str] = None):
+def get_variables(profile=None):
     """
     Gather and return all variables to robot
 
-    :param arg: path to directory with profiles
-    :param profile: name of profile
+    :param profile: path to cluster profile
+    :type profile: str
     :return: dict[str, Any] -- values for robot
     """
-    # Build default path to profiles directory
-    if not arg:
-        arg = os.getenv(PATH_TO_PROFILES_DIR)
-        if not arg:
-            arg = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', 'deploy', 'profiles'))
 
-    # load Profile
-    if not profile:
-        profile = os.getenv(PROFILE_ENVIRON_KEY)
-        if not profile:
-            raise Exception('Cannot get profile - {} env variable is not set'.format(PROFILE_ENVIRON_KEY))
+    # load Cluster Profile
+    profile = profile or os.getenv(CLUSTER_PROFILE)
 
-    profile = os.path.abspath(os.path.join(arg, '{}.yml'.format(profile)))
+    if not profile:
+        raise Exception('Can\'t get profile at path {}'.format(profile))
```

Comment:
It looks like it is old exception message. Message should be like "Cluster profile variable not set. Please set it or use env. variable {CLUSTER_PROFILE}"

Reply:
the idea of this message is to check if the profile argument has been really passed, the second try is to check if the profile presents on file system.

Reply:
But as I think it's better to use `{!r}` notation in this case